Commands/ContentTypes/SetDefaultContentTypeToList.cs

```diff
@@ -15,10 +15,10 @@ namespace SharePointPnP.PowerShell.Commands.ContentTypes
         SortOrder = 1)]
     public class SetDefaultContentTypeToList : SPOWebCmdlet
     {
-        [Parameter(Mandatory = true)]
+        [Parameter(Mandatory = true, HelpMessage = "The list object where the content type needs to be added")]
```

Comment:
Better: The name of the list, its ID or an actual list object from where the content type needs to be removed from.